app/src/main/res/values/dimens.xml

```diff
@@ -540,7 +540,7 @@
 
   <dimen name="promoted_story_card_layout_margin_end">8dp</dimen>
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
-  <dimen name="promoted_story_card_layout_width">280dp</dimen>
+  <dimen name="promoted_story_card_layout_width">280sp</dimen>
```

Comment:
Please keep these two values `promoted_story_card_width ` and `promoted_story_card_layout_width ` close to each other.

Maybe just shift `promoted_story_card_width ` above this `promoted_story_card_layout_width `
Also add a comment mentioning why these values are in sp.

Reply:
Done